src/test/scala/com/exasol/cloudetl/bucket/DeltaFormatBucketTest.scala

```diff
@@ -0,0 +1,110 @@
+package com.exasol.cloudetl.bucket
+
+import com.exasol.cloudetl.DummyRecordsTest
+import com.exasol.cloudetl.source.Source
+import com.exasol.cloudetl.storage.FileFormat
+
+import org.apache.spark.sql.SparkSession
+
+@SuppressWarnings(Array("org.wartremover.warts.MutableDataStructures"))
```

Comment:
I also have this SuppressWarning. Time to disable it or two cases are not enough?

Reply:
Let's wait for 3rd case :)